src/content/docs/reference/vscode.mdx

```diff
@@ -1,140 +1,310 @@
+### Defining projects
 
-```json title="settings.json"
-{
-	"editor.defaultFormatter": "<other formatter>",
-	"[javascript]": {
-		"editor.defaultFormatter": "biomejs.biome"
-	}
-}
-```
+Projects can be defined in two places:
+
+- At the **workspace** level (`.code-workspace`)
+- At the **workspace folder** level (`.vscode/settings.json`)
+
+:::note[Implicit projects]
+The extension automatically creates a project at the root of every workspace folder that does not have explicitly defined projects.
+:::
+
+#### Defining projects in a workspace
+
+To define projects in a **workspace**, add a `settings.biome.projects` property to the `.code-workspace` file. This setting accepts an
+array of project definitions, each of which has the following properties:
+
+- `name`: The **name of the workspace folder** that the project is associated with
+- `path` : The **path to the project directory**, _relative to the root of the workspace folder_.
+
+Please refer to the [`biome.projects` config reference](#biomeprojects) for more details.
+
+#### Defining projects in a workspace folder
+
+To define projects in a **workspace folder**, add a `biome.projects` property to the `.vscode/settings.json` file. This setting accepts an
+array of project definitions, each of which has the following properties:
+
+- `path`: The **path to the project directory**, _relative to the root of the workspace folder_.
+
+Please refer to the [`biome.projects` config reference](#biomeprojects) for more details.
+
+### Enabling and disabling
+
+The extension is always enabled because it needs to keep track of the state of the workspace, however the it provides two settings
+to control whether Biome LSP sessions should be created for indidividual projects:
+
+- The [`biome.enabled`](#biomeenabled) setting, which can be set globally and per-project.
+- The [`biome.requireConfigFile`](#biomerequireconfigfile) setting, which can be set globally and per-project.
+
+#### Using the `biome.enabled` setting
+
+As mentioned above, the [`biome.enabled`](#biomeenabled) setting can be set globally and per-project. When setting it globally, it applies to all workspace folders, unless
+they themselves override the setting. This allows you to enable or disable the extension for all workspace folders, or selectively enable or disable it for
+specific workspace folders. 
+
+#### Using the `biome.requireConfigFile` setting
+
+The [`biome.requireConfigFile`](#biomerequireconfigfile) setting can be set globally and per-project. When setting it globally, it applies to all workspace folders, unless
+they themselves override the setting. This allows you to require the presence of a [Biome configuration file](/guides/configure-biome#configuration-file-structure) for a Biome project to be enabled. 
+
+It is a good idea to enable this setting globally when you know you always use a `biome.json` file, and you want to avoid the extension creating projects
+for workspace folders that don't use Biome. 
```

Comment:
```suggestion
The [`biome.requireConfigFile`](#biomerequireconfigfile) setting can be set globally and per project. When setting it globally, it applies to all workspace folders unless
they override the setting. This allows you to require the presence of a [Biome configuration file](/guides/configure-biome#configuration-file-structure) for a Biome project to be enabled. 

It is a good idea to enable this setting globally when you know you always use a `biome.json` file, and you want to avoid the extension creating projects
for workspace folders that don't use Biome. 
```